README.md

```diff
@@ -268,23 +268,23 @@ Configuration documentation can be found at the [Instaclustr Terraform Registry]
 
```

Comment:
You'll need to bump the version in the makefile to be able to deploy this

Reply:
updated the version.